site/src/pages/components/selectlist.mdx

```diff
@@ -165,18 +165,40 @@ This example has a `<listbox>` which enables it to put arbitrary content into th
 ```html
 <selectlist>
   <listbox>
-    <div>Heading</div>
-    <div>First two options:</div>
-    <option>one</option>
-    <option>two</option>
-    <div>Second two options:</div>
-    <option>three</option>
-    <option>four</option>
+    <div class=container>
+      <div>
+        <div role=group>1-2</div>
+        <option>one</option>
+        <option>two</option>
+      </div>
+      <div>
+        <div role=group>3-4</div>
+        <option>three</option>
+        <option>four</option>
+      </div>
+      <div>
+        <div role=group>5-6</div>
+        <option>five</option>
+        <option>six</option>
+      </div>
+      <div>
+        <div role=group>7-8</div>
+        <option>seven</option>
+        <option>eight</option>
+      </div>
+    </div>
   </listbox>
 </selectlist>
 <style>
-div {
-  font-weight: bold;
+.container {
```

Comment:
Note: if you change to `<optgroup>` you can still do exactly this in CSS.

Reply:
yes.  these should be `optgroup` and either use the `label` attribute or the proposed `legend` element to provide the groups the accName.

Otherwise, if using divs, we need to correct the markup.

so these should be either:

```
<optgroup>
  <legend>name for group</legend>
  <option>...</option>
   ....
</optgroup>
```

or

```
<optgroup label='name for group'>
  <option>...</option>
   ...
</optgroup>
```

or

```
<div role=group aria-labelledby=f>
  <div id=f>name for group</div>
  <option>...</option>
   ....
</optgroup>
```

Reply:
ok i changed them to optgroups